wepool-app/src/screens/StartScreens/StartScreen.tsx

```diff
@@ -42,7 +42,9 @@ export const StartScreen = ({
     <View style={styles.container}>
       <Logo />
       <Header text="Welcome!"></Header>
+      {/* Comment the following line to deactivate login */}
```

Comment:
I changed this so that we don't deactivate the login functionality on the main branch.